pkg/vm/engine/disttae/types.go

```diff
@@ -498,7 +498,8 @@ type txnTable struct {
 	oldTableId uint64
 
 	// process for statement
-	proc *process.Process
```

Comment:
Help me understand WHY proc belongs to TABLE?

Reply:
in fact,  disttae.Transaction  owns the proc pointer ,  then it will be borrowed to distate.txnTable which be visisted by txn.